### ISSUE TITLE
Setting the command timeout with the latest .NET SqlClient

### ISSUE THREAD

**Issue by ErikEJ** [ErikEJ/erikej.github.io#22]
https://erikej.github.io/sqlclient/2020/10/26/sqlclient-commandtimeout-preview.html

**Comment by Piedone** [comment]
Quite cool, and thanks, didn't find this anywhere else despite looking for something like it for quite a while. And BTW I'd suggest adding a link to the blog post into the bug description.

**Comment by ErikEJ** [comment]
@Piedone  
>> And BTW I'd suggest adding a link to the blog post into the bug description.

Which bug description ??

**Comment by Piedone** [comment]
This here.

**Comment by SIEDA-EliasHeydrich** [comment]
We are on "System.Data.SqlClient" in version "4.8.3", in a "raw" Net6 environment.
As far as I can tell, the feature discussed in this blog post does not exist, at least it does not exist anymore.

Microsoft's docs do not mention this setting (see https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring?view=dotnet-plat-ext-6.0).

Trying it out anyway returns `Keyword not supported: 'command timeout'`.

**In short, this blog post should be removed for it is misleading in 2022.**

**Comment by ErikEJ** [comment]
@SIEDA-EliasHeydrich no it is not wrong  it refers to Microsoft.Data.SqlClient  - the modern replacement for System.Data.SqlClient

**Comment by SIEDA-EliasHeydrich** [comment]
Hehehe, alright, ignore my comment ;-)

Guess this happens when one stops reading all the Foo.Bar.Prefixing before the hundreds of Microsoft-Libraries.

**Comment by rahulsingii** [comment]
Thank you!

**Comment by developerpravin18** [comment]
it's worked.

thanks! :)
